Tests/SwiftStdlibTests/ArrayExtensionsTests.swift

```diff
@@ -8,14 +8,25 @@
 import XCTest
 @testable import SwifterSwift
 
-private struct Person: Equatable {
+private struct Person: Equatable, Hashable {
```

Comment:
Remove `Hashable`, you don't need it and then you won't need to implement the `hash(into:)` function.

Reply:
`Location` is not `Hashable`, that's why `func hash(into hasher: inout Hasher)` cannot be generated for `Person`.

Reply:
You don't need `Person` to be `Hashable`, so you can remove it.

Reply:
Got you. Done.